apiconcepts/integration/what_you_can_do_with_the_integration_API.md

```diff
@@ -3,10 +3,11 @@ What you can do with the Integration API
 
 This page provides a quick overview of what you can do with the <Var:ProductName> Integration API.
 
-The <Var:ProductName> Integration API enables third-party developers to:
+As you can see in the diagram below, the <Var:ProductName> Integration API enables third-party developers to provide custom plug-ins in order to:
 
 * [Extend or customize the user interface for Studio applications.](user_interface_integration.md)
 * [Create custom functionalities for the <Var:ProductName> application.](studio_automation.md)
 
+![PluginTypes](images/PluginTypes.png)
 
-This documentation features a number of usage examples that illustrate how to develop against the <Var:ProductName> Integration API. All provided examples are written in C#.
\ No newline at end of file
+This documentation features a number of usage examples that illustrate how to develop against the <Var:ProductName> Integration API. All provided examples are written in C#.
```

Comment:
Please change this also to:
<Var... > API documentation features examples on common and best practices when developing plug-ins using <var ..> API. All provided...